src/components/Image/Image.tsx

```diff
@@ -81,7 +82,7 @@ const Image = (props: ImageProps): JSX.Element => {
   return (
     <>
       <img
-        className={`bsds-image${ratioClasses}${className ? ' ' + className : ''}`}
+        className={`bsds-image${ratioClasses} ${className || ''}`}
```

Comment:
in some cases, our bsds-mt-#x util classes are getting overridden by component classes. we'll likely need to add !important or look at where we're setting margins on the outermost wrappers for our components. we can create a follow up ticket after this PR is merged.